iceoryx2-ffi/cxx/include/iox2/log_level.hpp

```diff
@@ -18,12 +18,12 @@
 namespace iox2 {
 
 enum class LogLevel : uint8_t {
-    TRACE = 0,
-    DEBUG = 1,
-    INFO = 2,
-    WARN = 3,
-    ERROR = 4,
-    FATAL = 5,
+    Trace = 0,
```

Comment:
@elfenpiff Since this is C++ shouldn't the C++ convention be followed i.e. SCREAMING_CASE for enums? Don't care either way, just curious of the convention. I had thought to stick to the language-specific conventions.

Reply:
No, for C++ enums we decided to follow the rust naming conventions.